airflow/www/views.py

```diff
@@ -528,8 +528,9 @@ def code(self, session=None):
                 code = f.read()
             html_code = highlight(
                 code, lexers.PythonLexer(), HtmlFormatter(linenos=True))
-        except OSError as e:
-            html_code = str(e)
+        except Exception as e:
+            html_code = 'Failed to load file ' + (dag.fileloc if dag else 'unknown') +\
+                        '.<br/> Details: ' + str(e)
```

Comment:
This is a possible HTML escaping vulnerability here. Do not pass untrusted user input of any form to the UI without HTML escaping it.

Reply:
True. I did not think about it. thanks for spotting!